test/micro/org/openjdk/bench/java/lang/StringBuilders.java

```diff
@@ -69,6 +71,13 @@ public void setup() {
         sbLatin2 = new StringBuilder("Latin1 string");
         sbUtf16 = new StringBuilder("UTF-\uFF11\uFF16 string");
         sbUtf17 = new StringBuilder("UTF-\uFF11\uFF16 string");
+        int size = 16;
+        intsArray = new int[size];
+        longArray = new long[size];
```

Comment:
```suggestion
        intArray = new int[size];
        longArray = new long[size];
```
or
```suggestion
        intsArray = new int[size];
        longsArray = new long[size];
```
Either both plural, or both singular